packages/realm-server/middleware/index.ts

```diff
@@ -94,13 +94,18 @@ export function rootRealmRedirect(
 ): (ctxt: Koa.Context, next: Koa.Next) => void {
   return (ctxt: Koa.Context, next: Koa.Next) => {
     let url = fullRequestURL(ctxt);
+
+    let realmUrlWithoutQueryParams = url.href.split('?')[0];
     if (
-      !url.href.endsWith('/') &&
+      !realmUrlWithoutQueryParams.endsWith('/') &&
```

Comment:
Could this use `url.pathname.endsWith('/')` instead of the handcrafted local variable?

Reply:
probably not, i think the reverse resolution logic below needs the beginning part of the URL